phpunit.xml

```diff
@@ -14,12 +14,16 @@
             <directory>tests/unit</directory>
         </testsuite>
     </testsuites>
-    <filter>
-        <whitelist>
+    <coverage>
+        <include>
             <directory suffix=".php">src/</directory>
-        </whitelist>
-    </filter>
+        </include>
+        <report>
+            <html outputDirectory="build/coverage"/>
+            <text outputFile="build/coverage.txt"/>
+        </report>
+    </coverage>
     <logging>
-        <log type="coverage-html" target="build/coverage" charset="UTF-8" yui="true" highlight="true"/>
+        <junit outputFile="build/report.junit.xml"/>
```

Comment:
You've changed the output, why?

Reply:
Because it changed when upgrading phpunit

Reply:
Remove the `<logging>` node, add the `<coverage>` node:

```
<coverage>
    <include>
      <directory suffix=".php">src/</directory>
    </include>
    <report>
      <html outputDirectory="./build/coverage/"/>
    </report>
  </coverage>
```

This will result in the same as the previous config.